Store/DAL/StoreContext.cs

```diff
@@ -10,7 +10,7 @@ public StoreContext(DbContextOptions<StoreContext> options)
         {
             Database.EnsureCreated();
         }
-
+        
```

Comment:
changes with this looks like not so clean. before push or before PR try to use some kind of Prettier. in VS if i'm not mistaken shortkey is Ctrl+K+D. in Rider Ctrl+Alt+L.